app/components/candidate_interface/course_choices_review_component.rb

```diff
@@ -151,7 +151,7 @@ def start_date_row(application_choice)
     end
 
     def interview_row(application_choice)
-      if application_choice.interviews.present?
+      if application_choice.interviews.kept.any?
```

Comment:
:+1: `any?` is better (faster) than `present?`. Apparently `exists?` is even better but probably won't make a measurable difference here because interviews is a small result set.